elasticsearch/manifests/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - elasticsearch.yaml
+
+patches:
+  - target:
+      kind: StatefulSet
+      name: elasticsearch-master
+    patch: |-
+      # replace ELASTICSEARCH_CLUSTER_HOSTS
+      # but first, check if this index contains correct env
+      - op: test 
+        path: /spec/template/spec/containers/0/env/7/name
+        value: "ELASTICSEARCH_CLUSTER_HOSTS"
+      - op: replace
+        path: /spec/template/spec/containers/0/env/7/value
+        value: "elasticsearch-master-hl,"
+      # replace ELASTICSEARCH_ADVERTISED_HOSTNAME
+      # but first, check if this index contains correct env
+      - op: test 
+        path: /spec/template/spec/containers/0/env/11/name
+        
+        value: "ELASTICSEARCH_ADVERTISED_HOSTNAME"
+      - op: replace
+        path: /spec/template/spec/containers/0/env/11/value
+        value: "$(MY_POD_NAME).elasticsearch-master-hl"
```

Comment:
why do we change these two vars, what's wrong with the original ones?

Reply:
They contain the namespace in the value initially. I just looked at that.

Reply:
> They contain the namespace in the value initially. I just looked at that, as I was wondering the same thing

:+1: might be helpful to have a comment here to help anyone else in the future with the same question

Reply:
Added comment [here](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/16/commits/5ef645f9a43d86f54f5812d919014d542f7976ae).